package.json

```diff
@@ -29,6 +29,7 @@
     "angular-gettext": "2.3.4",
     "angular-gettext-tools": "2.2.3",
     "angular-messages": "1.5.8",
+    "slug": "0.9.1",
```

Comment:
It's in the dependencies of ``angular-slug`` https://github.com/lucasconstantino/angular-slug/blob/master/package.json#L15
Shall we also add it to our own dependencies?